step1-clean/query.py

```diff
@@ -0,0 +1,16 @@
+from db import session, SQLImage
+
+def filterThumbnails():
+  query = session.query(SQLImage).filter_by(isThumbnail="True").all()
+
+  for row in query:
+    print(row)
+
+
+def queryAll():
+  query = session.query(SQLImage).all()
+  for row in query:
+    print(row)
+
+
+filterThumbnails()
```

Comment:
please reread the instructions. this function does not correct

Reply:
Oh okay, I see. I think I was looking at the gdoc for this part. Is the github going to be the official place for instructions and the gdoc more of a draft/ideas?

Reply:
yeah the gdoc is a brainstorming doc used to help me outline for detailed instructions on github

Reply:
Okay, got it. Not sure if you noticed, but 00C-Synthesis appears to be a mirror copy of 00B-Persisting to DB.

Reply:
thanks for catching that. forgot to push it